bundles/org.openhab.binding.lutron/README.md

```diff
@@ -289,7 +331,7 @@ Therefore, the channel state will only transition OFF->ON->OFF when you send a O
 
 For maintained CCOs, sending ON and OFF commands works as expected, and the channel state updates as expected when either openHAB commands or external events change the CCO device state.
 
-Example:
+Thing configuration file example:
```

Comment:
Why did you change all these?

Reply:
Just to clarify that those examples only include excerpts from thing files.  I had updated all of the examples in a previous commit and forgot to make that change.  So I sort of snuck it in here. The README has been a work in progress.